lib/date_time/Kconfig

```diff
@@ -9,6 +9,13 @@ menuconfig DATE_TIME
 
 if DATE_TIME
 
+config DATE_TIME_AUTO
+	bool "Automatically update time"
+	select AT_MONITOR
+	default y
+	help
+		Update Zephyr and modem time based on available time sources.
+
 config DATE_TIME_UPDATE_INTERVAL_SECONDS
 	int "Date time update interval, in seconds"
```

Comment:
Depending on the drifting of the clock, periodical updates are most likely necessary. I'm not sure how this has currently been implemented, but this should take into account also updates from the modem. I.e. don't use NTP for time update if we have already got an update from the modem.